readthedocs/builds/models.py

```diff
@@ -698,3 +699,114 @@ def run_time(self):
         if self.start_time is not None and self.end_time is not None:
             diff = self.end_time - self.start_time
             return diff.seconds
+
+
+class VersionAutomationRule(PolymorphicModel, TimeStampedModel):
+
+    """Versions automation rules for projects."""
+
+    REGEX_RULE = 'regex-rule'
+    RULE_TYPES = (
+        (REGEX_RULE, _('Regular expression rule')),
+    )
+
+    ACTIVATE_VERSION_ACTION = 'activate-version'
+    ACTIONS = (
+        (ACTIVATE_VERSION_ACTION, _('Activate version on match')),
+    )
+
+    project = models.ForeignKey(
+        Project,
+        related_name='automation_rules',
+        on_delete=models.CASCADE,
+    )
+    priority = models.IntegerField(_('Rule priority'))
```

Comment:
I assume that 0 is max priority, right? Maybe a comment or `help_text` for the user here will be helpful.

Reply:
I was thinking 1 :laughing: